index.html

```diff
@@ -0,0 +1,103 @@
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="keywords" content="HTML, CSS, Javascript">
+    <meta name="author" content="Maen Kh">
+    <title>Global Warmming</title>
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="components/header.js"  defer></script>
+    <script src="components/footer.js"  defer></script>
+    <script src="/JS/javascript.js"></script>
+    
+    <link rel="stylesheet" href="/css/style.css">
+</head>
+<body>
+    <header-component></header-component>
+    <main class="main-section">
+        <a class="btn-hamburger" href="#">
+            <div class="main-header-hamburger">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </div>
+        </a>
+        <ul class="hamburger-menu">
+            <li class="ham-menu-items">
+                <a href="/about.html" class="ham-menu-link">About</a>
+            </li>
+            <li class="ham-menu-items">
+                <a href="/index.html" class="ham-menu-link">Program</a>
+            </li>
+            <li class="ham-menu-items">
+                <a href="#" class="ham-menu-link">Join</a>
+            </li>
+            <li class="ham-menu-items">
+                <a href="#" class="ham-menu-link">Sponsor</a>
+            </li>
+            <li class="ham-menu-items">
+                <a href="#" class="ham-menu-link">News</a>
+            </li>
+            <li class="ham-menu-items">
+                <a href="#" class="ham-menu-link">CC Campaign</a>
+            </li>
+        </ul>
+        <h1 class="main-section-title">"Hello! Green World"</h1>
+        <h2 class="main-section-subtitle ">SAVING WORLD IS EVERYBODY RESPONSIBILITY</h2>
+        <span class="main-section-desc">It is clear that climate change is an all encompassing challenge, impacting all spheres of life and every region of the world. Even with the commitments made at COP26 to phase down coal, cut methane emissions and halt deforestation, some reports suggest the world is still on track for 2.4C of warming by 2030. </span>
+        <div class="main-section-date">
+            <h2>8 - 10 March 2022</h2>
+            <p>
+                @Accelerating Action to Achieve Net-Zero <br>
+                In-Person & Digital | Four Seasons Park Lane, London | #FTClimateCapital
+            </p>
+        </div>
+    </main>
+    <section class="main-program">
+        <h1 class="main-program-title">Main Program</h1>
+        <div class="main-program-underline"></div>
+        <ul>
+            <li class="main-program-sections">
+                <img class="main-program-sections-logo" src="/assets/images/lecture.svg" alt="lecture">
```

Comment:
- [ ] When we hover the mouse over cards a white border around cards should appear as on the design, Please apply hover effects to your cards so the cards show a white border when we hove mouse over them.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/157689981-91c089cf-d233-49a4-b0d5-3e74f7a6d17b.png)

Reply:
Thank you for your full constructive feedback,
I really appreciate your complements 😄 
of course, I will do the suggested modification and then merge the project.
thanks again 😃